webapp/src/Command/ImportEventFeedCommand.php

```diff
@@ -953,7 +953,7 @@ protected function importTeam(array $event)
                 $this->em->flush();
                 $this->eventLogService->log('teams', $team->getTeamid(),
                                             EventLogService::ACTION_DELETE,
-                                            $this->contestId, null, $team->getExternalid());
+                                            $this->contestId, null, $team->getTeamid());
```

Comment:
not `getIcpcId`?

Reply:
Nope, this was a bug: the team ID in the event feed is *always* our `teamid`, never the `icpc_id`.